patches/chrome-browser-resources-settings-site_settings_page-site_settings_page.html.patch

```diff
@@ -28,15 +28,24 @@ index b2894017d199903bb468124871b0a4d1223f923f..84324a86432dc7f47186da91931764d7
      <div id="javascript" class="settings-box two-line"
          category$="[[ContentSettingsTypes.JAVASCRIPT]]"
          data-route="SITE_SETTINGS_JAVASCRIPT" on-click="onTapNavigate_"
-@@ -242,6 +261,7 @@
+@@ -221,6 +240,7 @@
+             aria-describedby="popupsSecondary"></button>
+       </paper-icon-button-light>
+     </div>
++<if expr="_google_chrome">
+     <template is="dom-if" if="[[enableSafeBrowsingSubresourceFilter_]]">
+       <div id="ads" class="settings-box two-line"
+           category$="[[ContentSettingsTypes.ADS]]"
+@@ -242,6 +262,8 @@
          </paper-icon-button-light>
        </div>
      </template>
++</if>
```

Comment:
can we merge the `/if` and `if google_chrome` here?

Reply:
Was intentionally separate here because then if something is added in between it won't be hidden by the patch